### PR TITLE
link to wynton.ucsf.edu fixed

### PR DESCRIPTION
**Pull request by pavgup**
link was accidentally relative. also, the https connection is using a self-signed cert...

### CODE REVIEW

**Comment by pavgup** [comment]
@HenrikBengtsson, I definitely see that you assigned the PR to me, but alas, that's all I see..

**Comment by HenrikBengtsson** [comment]
You don't see my review comment "Since it's self-certified now, I think we should point to http://wynton.ucsf.edu/ instead." above (GH online), but I'd expect a notification to also go out?

**Comment by pavgup** [comment]
@HenrikBengtsson, as crazy as it seems, I can't seem to see that. On the self-signed question, why not just grab a cert and call it a day? Lets be a part of the "future of the web" (haha): https://developers.google.com/web/fundamentals/security/encrypt-in-transit/why-https

**Comment by HenrikBengtsson** [comment]
> ... why not just grab a cert and call it a day?

I'm not in charge/control of wynton.ucsf.edu

**Comment by HenrikBengtsson** [comment]
Regarding code review comments: This is what it looks (see bottom) like for me in a browser.  Don't you see the same?

screenshot from 2018-02-08 15-31-26

**Comment by pavgup** [comment]
Weird, I really can't see that comment. Maybe it's some kind of permissions thing? Could also be me just not knowing. And dropped the `s` -- should link to the wynton.ucsf.edu in the clear

File: docs/_layouts/default.html [hunk 1]
```diff
@@ -160,7 +160,7 @@
      </span>
 <p class="muted credit" style="font-size: inherits;">
 <a href="{{ '/' | relative_url }}">HPC Wynton</a>
-(<a href="{{ '/' | relative_url }}wynton.ucsf.edu/index.html">front</a>)
+(<a href="https://wynton.ucsf.edu/index.html">front</a>)
```
[the remark this answers is not in the document]
Comment (pavgup) [reply docs/_layouts/default.html]: I see it now :)

[the remark this answers is not in the document]
Comment (pavgup) [reply docs/_layouts/default.html]: And it does trigger an email -- a good thing, I'd think. Now, when do you integrate slack ;)

[the remark this answers is not in the document]
Comment (HenrikBengtsson) [reply docs/_layouts/default.html]: Great.  So, the error was on my end.

**Pull request merged**